growth/too/tasks/ztf_client.py

```diff
@@ -89,3 +93,72 @@ def ztf_references():
                          reference_filter_ids=rows['fid'].tolist(),
                          reference_filter_mags=rows['maglimit'].tolist()))
     models.db.session.commit()
+
+
+@celery.task(base=PeriodicTask, shared=False, run_every=3600)
+def ztf_depot(start_time=None, end_time=None):
```

Comment:
Please add a doctstring to explain what this function does.

Reply:
added!

Reply:
This docstring is not illuminating for me.

Reply:
ok i expanded it a bit.